src/main/java/RegisterPage.java

```diff
@@ -0,0 +1,273 @@
+        fillName(name);
+        fillRegisterEmail(email);
+        return this ;
+    }
+
+    public RegisterPage clickOnSignUpButton(){
+        driver.element().click(submitSignupButton);
+        return this ;
+    }
+
+    public RegisterPage clickOnMale() {
+        driver.element().click(genderMale);
+        return this;
+    }
+
+    public RegisterPage verifyAccountInfoTitle() {
+        driver.element().verifyThat(accountInfoTitle).isVisible();
+        return this;
+    }
+
+    public RegisterPage clearInfoName(){
+        driver.element().clear(infoName);
+        return this ;
+    }
+
+    public RegisterPage fillInfoName(String name){
+        driver.element().type(infoName,name);
+        return this ;
+    }
+    public RegisterPage fillInfoPassword(String password){
+        driver.element().type(registerPassword,password);
+        return this ;
+    }
+    public RegisterPage fillInfoData (String name, String password){
+        clearInfoName();
+        fillInfoName(name);
+        fillInfoPassword(password);
+        return this ;
+    }
+
+
+    public RegisterPage verifyInfoEmail(String email) {
+        driver.element().verifyThat(infoEmail).attribute("Value").isEqualTo(email);
+        return this;
+    }
+
+    public RegisterPage fillDay(){
+        driver.element().click(infoDay);
+        driver.element().click(birthDay);
+        return this ;
+    }
+
+    public RegisterPage fillMonth(){
+        driver.element().click(infoMonth);
+        driver.element().click(birthMonth);
+        return this ;
+    }
+    public RegisterPage fillYear(){
+        driver.element().click(infoYear);
+        driver.element().click(birthYear);
+        return this ;
+    }
+
+    public RegisterPage fillBirthDate (){
+        fillDay();
+        fillMonth();
+        fillYear();
+        return this ;
+    }
+
+    public RegisterPage clickOnNewsletterCheckBox() {
+        driver.element().click(newsLetterCheckBox);
+        return this ;
+    }
+
+    public RegisterPage clickOnReceiveOffersCheckBox() {
+        driver.element().click(offersCheckBox);
+        return this ;
+    }
+
+    public RegisterPage fillFirstName(String firstN) {
+        driver.element().type(firstNameField, firstN);
+        return this ;
+    }
+
+    public RegisterPage fillLastName(String lastN) {
+        driver.element().type(lastNameField,lastN);
+        return this ;
+    }
+
+    public RegisterPage fillCompany(String company) {
+
+        driver.element().type(companyField,company);
+        return this ;
+    }
+
+    public RegisterPage fillAddress(String addressOne) {
+        driver.element().type(addressOneField,addressOne);
+        return this ;
+    }
+
+    public RegisterPage fillAddressTwo(String addressTwo) {
+        driver.element().type(adressTwoField,addressTwo);
+        return this ;
+    }
+    public RegisterPage fillAddressData(String firstN, String lastN,
```

Comment:
firstN , lastN ?? what is the meaning of them ?

Reply:
this comment was resolved